lib/plugins/aws/deploy/index.js

```diff
@@ -32,8 +33,9 @@ class AwsDeploy {
     Object.assign(
       this,
       extendedValidate,
-      createStack,
       setBucketName,
+      shouldBeDeployed,
```

Comment:
Somehow I disagree with the naming - all other methods are named after the actions they do. `shouldBeDeployed` doesn't really fit in there. Maybe `checkForDifferences` would be more appropriate.